src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-credentialsId.html

```diff
@@ -1,11 +1,13 @@
 <div>
-    <p>When running a job, Jenkins will use these credentials to check out the source code for builds. For Jenkins to
-        do this, it needs credentials with access to the projects and repositories you want it to build from.</p>
+    <p>When running a job, Jenkins will use these credentials to check out the source code for builds. For Jenkins to do
+        this, it needs credentials with access to the projects and repositories you want it to build from.</p>
 
     <p>To provide Jenkins with credentials you can:</p>
     <ul>
-        <li>Choose credentials from the list.</li>
-        <li>Add credentials as a username and password.</li>
+        <li>choose credentials from the list</li>
+        <li>add credentials as a username and password (for the password, you can enter a Bitbucket Server password or a
```

Comment:
```suggestion
        <li>add credentials as a <strong>Username with password</strong> (for the password, you can enter a Bitbucket Server password or a
```
? so we are consistent with our other documentation?